web/business-registry-dashboard/app/stores/affiliations.ts

```diff
@@ -463,7 +463,7 @@ export const useAffiliationsStore = defineStore('brd-affiliations-store', () =>
         }
       }
     },
-    { debounce: 400 } // 400ms debounce time - wait for user input to settle
+    { debounce: 500 } // 500ms debounce time - wait for user input to settle
```

Comment:
Is this enough time between characters to type in a business name, business id or NR number? I expected this would be larger. I recommend checking with UX team (J/J) for their opinion on this (and they can test using the preview URL).

Reply:
After some testing, I am OK with the 500ms delay. Did Jacqueline provide any extra feedback?

Anyway, I'm resolving this conversation.